pkg/vectorstore/chromem/chromem.go

```diff
@@ -58,7 +58,7 @@ func (s *Store) AddDocuments(ctx context.Context, docs []vs.Document, collection
 		return nil, vs.ErrCollectionNotFound{Collection: collection}
 	}
 
-	err := col.AddDocuments(ctx, chromemDocs, runtime.NumCPU()/2)
+	err := col.AddDocuments(ctx, chromemDocs, int(math.Ceil(float64(len(chromemDocs))/20.0)))
```

Comment:
Can we make this configurable?

Reply:
Good idea! will do

Reply:
Addressed as an environment variable. Also, I make this a fix value to be the number of goroutinues for user to understand it better.